exercise-5.js

```diff
@@ -0,0 +1,35 @@
+function crypto_calculator(money, bitcoin_price, ethereum_price, litecoin_price) {
+    if (money <= 0) {
+        console.log("Insufficient funds");
+        return [0, 0, 0];
+    }
+
+    let bitcoin_max, ethereum_max, litecoin_max = 0;
+
+    function coin_max(price, max) {
+        if (price < 0) {
+            max = undefined;
+        } else if (price == 0) {
+            max = 0;
+        } else {
+            max = Math.floor(money / price);
+        }
+        return max; // returns the value but doesn't store it in a variable. Make sure to do that yourself
+    }
```

Comment:
For example, try multiple returns here

Reply:
Same return statement multiple times?

Reply:
no no, in each of the conditions have a return statement

Reply:
in each condition have the same return statement?

Reply:
i got it